packages/client/config-overrides.js

```diff
@@ -0,0 +1,40 @@
+const { override } = require("customize-cra")
+const cspHtmlWebpackPlugin = require("csp-html-webpack-plugin")
+
+const cspConfigPolicy = {
+  "default-src": ["'none'"],
+  "script-src": [],
+  "style-src": ["'self'", "'unsafe-inline'"],
+  "img-src": ["'self'", "data:", "cid:", "mid:"],
+  "child-src": ["data:"],
+  "font-src": ["'self'"],
+  "media-src": ["'self'", "data:", "cid:", "mid:"],
+  "object-src": ["'none'"],
+  "form-action": ["'none'"]
+}
```

Comment:
The packaged version of the app loads html, css, and javascript assets via the `file:` protocol instead of `http:`. Browsers treat these protocols differently in a few ways. In particular using a content security policy such as `'self'` does not work well with `file:` assets. We need to switch to a more secure nonce policy, where the browser will only execute a script tag if it has a `nonce` attribute with a value that is listed in the content security policy. Script tags are written by Webpack, so we need to modify the Webpack configuration to include nonces; and because the client package uses create-react-app, we need to use tools such as `customize-cra` to patch create-react-app's Webpack configuration.